package.json

```diff
@@ -13,6 +13,7 @@
     "reshape": "^0.4.0",
     "reshape-loader": "^0.4.0",
     "rooftop-client": "^0.2.1",
+    "spike-util": "^0.4.0",
```

Comment:
This is and old, pre-1.0 release. If you use the latest version it should help: https://github.com/static-dev/spike-util/releases